app/Services/UserService.php

```diff
@@ -51,4 +51,50 @@ public function createUser($request)
             return false;
         }
     }
+
+    /**
+     * @param $id
+     * @return UserRepository
+     */
+    public function getUserById($id)
+    {
+        return $this->userRepository->find($id);
+    }
+
+    /**
+     * @param $id
+     * @param $request
+     * @return bool
+     */
+    public function updateUser($id, $request)
```

Comment:
Chỗ này nếu không dùng Request riêng nào thì nên để là
```suggestion
    public function updateUser($id, Request $request)
```

Reply:
au ke e